src/MediaPlayer/MediaPlayer.cs

```diff
@@ -216,6 +216,33 @@ public enum MPMusicShuffleMode : nint {
 
 	public delegate void MPMediaItemEnumerator (string property, NSObject value, ref bool stop);
 
+	[iOS (3,0)]
```

Comment:
no need for them before 6.0, lack of attribute means always available in the OS we support